roles/kubernetes/node/templates/kubelet.kubeadm.env.j2

```diff
@@ -0,0 +1,56 @@
+# logging to stderr means we get it in the systemd journal
```

Comment:
Note that you will have to keep this up-to-date with what's in `build/debs/10-kubeadm.conf` per released kubeadm version.

Reply:
No surprise here

Reply:
Add that as a comment and link to the upstream source?